src/main/java/hw_5/myClass/MyInt.java

```diff
@@ -0,0 +1,15 @@
+package hw_5.myClass;
+
+public class MyInt {
+    public int x;
+
+    public MyInt(int x){
+        this.x = x;
+    }
+    @Override
```

Comment:
Please add extra space between 8 and 9 rows

Reply:
fixed